python/cudf/cudf/_cuda/gpu.pxd

```diff
@@ -0,0 +1,315 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+cdef extern from "cuda.h" nogil:
+    cdef enum cudaDeviceAttr:
```

Comment:
Sure would be nice if someone were to make a standard set of Python wrappers for the CUDA runtime APIs!

Reply:
There's a reason this is in `cudf/_cuda` as opposed to `cudf/_libxx` ;)